src/fragments/lib/graphqlapi/graphql-from-node.mdx

```diff
@@ -150,84 +147,60 @@ General information
 You can access the following resource attributes as environment variables from your Lambda function
 	API_<YOUR_API_NAME>_GRAPHQLAPIENDPOINTOUTPUT
 	API_<YOUR_API_NAME>_GRAPHQLAPIIDOUTPUT
-	API_<YOUR_API_NAME>_GRAPHQLAPIKEYOUTPUT
 ? Do you want to edit the local lambda function now? No
 ```
 
 The CLI will automatically configure the Lambda execution IAM role required by the Lambda function to call the GraphQL API. The following function will sign the request and use environment variables for the AppSync and Region that `amplify add function` created for you.
 
+Next, you need to install two npm packages for your AWS Lambda function. To do so, open your terminal in the `amplify/backend/function/<YOUR_FUNCTION_NAME>/src` directory in your newly created AWS Lambda function and run the command `npm install appsync-client graphql-tag`. Once the packages are installed, edit your index.js as the following:
```

Comment:
Next, you need to install two npm packages for your AWS Lambda function. To do so, open your terminal in the `amplify/backend/function/<YOUR_FUNCTION_NAME>/src` directory in your newly created AWS Lambda function and run the command `npm install appsync-client graphql-tag`. Once the packages are installed, add this your ```index.js```: